app/src/main/export.js

```diff
@@ -22,7 +22,7 @@ export const startExport = () => {
     if (isExportInProgress) {
       const buttonIndex = dialog.showMessageBox(exportWindow, {
         type: 'question',
-        buttons: ['Cancel Export', 'Continue'],
+        buttons: ['Cancel Export', 'Continue Export'],
         defaultId: 1,
```

Comment:
I think `cancel` should be the default, so you just can press enter

Reply:
Nice, updated!